dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -248,12 +248,9 @@ public TrackedEntity getTrackedEntity(
    *
    * @return the TE object if found and accessible by the current user
    * @throws NotFoundException if uid does not exist
-   * @throws ForbiddenException if TE registration org unit not in user's scope or not enough
-   *     sharing access
    */
   private TrackedEntity getTrackedEntity(
```

Comment:
Maybe it would make sense to explain in the name of the method or in the JavaDoc that this method is meant to be used only for relationships.